### PR TITLE
enable RHEL, Fedora, RHCOS, for sysctl ignition scripts

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

File: ocp4/profiles/moderate.profile [hunk 1]
```diff
@@ -131,6 +132,7 @@ selections:
     - grub2_vsyscall_argument.role=unscored
     - grub2_vsyscall_argument.severity=info
     - grub2_pti_argument
+    - sysctl_user_max_user_namespaces
```
Comment (jhrozek) [inline ocp4/profiles/moderate.profile:135]: Do we really want to disable user namespaces for a platform that does nothing but run containers?

File: ocp4/profiles/moderate.profile [hunk 2]
```diff
@@ -79,27 +79,28 @@ selections:
...
+    - sysctl_net_ipv4_conf_all_send_redirects
+    - sysctl_net_ipv4_conf_default_send_redirects
+    - sysctl_net_ipv4_conf_all_log_martians
+    - sysctl_net_ipv4_conf_default_log_martians
+    - sysctl_net_ipv4_conf_all_rp_filter
+    - sysctl_net_ipv4_conf_default_rp_filter
+    - sysctl_net_ipv4_icmp_ignore_bogus_error_responses
+    - sysctl_net_ipv4_icmp_echo_ignore_broadcasts
+    - sysctl_net_ipv4_tcp_syncookies
+    - sysctl_net_ipv4_ip_forward
```
Comment (jhrozek) [inline ocp4/profiles/moderate.profile:103]: Check with @JAORMX if these can be enabled alrady, I know they were causing issues in CI and at least some of the issues were caused by bad labels, but I don't know for certain if all the issues were found already.

**Comment by jhrozek** [comment]
btw more of my remediations were OCP-only, so I fixed the others in PR #5646

File: linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_all_accept_source_route/ignition/shared.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_ocp
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
```
Comment (yuumasato) [inline linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_all_accept_source_route/ignition/shared.yml:1]: Could you clarify what is the intent of making Ignition remediations applicable on RHEL and Fedora?

Comment (jhrozek) [reply linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_all_accept_source_route/ignition/shared.yml:1]: From another issue "These are still needed for RHEL worker nodes. Trying to be a good upstream OKD steward and added Fedora too. The ignition scripts won't get added to the RHEL and Fedora datastreams without these lines."

**Comment by shawndwells** [comment]
Closing this PR. Changes were made in smaller PRs to ease merge/review/

**Pull request closed without merging**